app/src/main/java/nl/eduvpn/app/entity/CurrentVPN.kt

```diff
@@ -0,0 +1,9 @@
+package nl.eduvpn.app.entity
+
+import com.wireguard.config.Config
+
+sealed class CurrentVPN
```

Comment:
I think `VpnProtocol` would be a more fitting name

Reply:
Fixed in https://github.com/eduvpn/android/pull/318/commits/3cd59bcbed42a20e85ff8b0b2a17b5afee371bab